### PR TITLE
Consistent use of stringify (for output) and hash (for hashing).

### PR DESCRIPTION
**Pull request by domoritz**

### CODE REVIEW

File: src/util.ts [hunk 1]
```diff
@@ -37,11 +37,25 @@ export function omit(obj: object, props: string[]) {
   return copy;
 }
 
-export function hash(a: any) {
+/**
+ * Converts any object into a string representation that can be consumed by humans.
+ */
+export function stringify(a: any): string {
```
[the remark this answers is not in the document]
Comment (domoritz) [reply src/util.ts]: Let's say you have something like `console.log('The value "${stringify(val)}" was unexpected')` then you get `The value ""foo"" was unexpected.`

[the remark this answers is not in the document]
Comment (kanitw) [reply src/util.ts]: I see.  I still don't like the fact that now the term `stringify` is overloaded.  There is the classic stringify and the stringify without quotes. 

It seems like this problem can be a fixed by just not adding extra quotes to the command
```
console.log(`The value ${stringify(val)} was unexpected`);
```

(The value "foo" is unexpected / The value 5 is unexpected / The value {"a": 1} is unexpected all read fine to me.)

[the remark this answers is not in the document]
Comment (domoritz) [reply src/util.ts]: Then we should do that consistently in our code, lol.

[the remark this answers is not in the document]
Comment (kanitw) [reply src/util.ts]: You can pivot this PR to fix those instead :)

[the remark this answers is not in the document]
Comment (domoritz) [reply src/util.ts]: Done

**Pull request merged**